pom.xml

```diff
@@ -85,8 +85,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-war</artifactId>
-      <!--to have access to User.getById-->
-      <version>1.651.2</version>
+      <version>2.60.3</version>
```

Comment:
As discussed in Gitter, I am about submitting a wider thread about dropping Java 7 support in our devtools

Reply:
Started https://groups.google.com/forum/#!topic/jenkinsci-dev/SdMOMKs7XIQ